packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-reporters/SpecReporter.tool.mdx

```diff
@@ -0,0 +1,170 @@
+# Spec Reporter
+
+The Spec Reporter writes test results to file in JSON format.
+
+## Installation
+
+The spec reporter should be installed as a development dependency.
```

Comment:
The spec reporter is part of the default wdio config. Should we document it as such? Do we ever expect people to use it by itself?

Reply:
Or maybe we focus the documentation on the default config and have this doc in an advanced section since i don't think the majority of our users will need it?

Reply:
There's a note below:

> Note: The spec reporter is enabled by default in the `terra-functional-testing` wdio configuration. If you are using/extending the default `wdio.conf.js` it is not necessary to reimport the reporter.

But I had similar concerns about the documentation. Distinguishing between a stand alone reporter and how to interact with the reporter when using the test runner and extending the wdio.confg.js.

I think we can either have two different pages of documentation or we rework this one to be from the perspective of the test runner

Reply:
I [reorganized](https://github.com/cerner/terra-toolkit/pull/535/commits/269b822d40792f8ccfc0bf7c8c35656298bb4345) the documentation, but then decided to just go ahead and remove the advanced section all together:

https://github.com/cerner/terra-toolkit/pull/535/commits/9db62fa3c3ead3ec0e28016e13a05b22c77486b2